lib/src/base-styles/_nx-colors.scss

```diff
@@ -55,9 +55,12 @@
   --nx-color-alert-warning: var(--nx-swatch-orange-50);
   --nx-color-alert-error: var(--nx-swatch-red-40);
 
-  //NxBinaryDonutChart variables
-  --nx-color-binary-donut-chart-background-fill: var(--nx-swatch-indigo-95);
-  --nx-color-binary-donut-chart-arc-stroke: var(--nx-swatch-teal-60);
+  --nx-color-progress-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-fill: var(--nx-swatch-teal-60);
+
+  //NxBinaryDonutChart - deprecated, use --nx-color-progress instead
```

Comment:
Deprecated variables get moved to the bottom

Reply:
> Deprecated variables get moved to the bottom

I believe we can actually just remove these variables instead of deprecating. They are not currently being used in the gallery, and were created when the dark mode ticket was done for NxBinaryDonutChart.

Reply:
> > Deprecated variables get moved to the bottom
> 
> I believe we can actually just remove these variables instead of deprecating. They are not currently being used in the gallery, and were created when the dark mode ticket was done for NxBinaryDonutChart.

Ah, none of the dark mode tickets has been merged with main so that means those NxBinaryDonutChart variables are definitely not being used by any project. I removed them here:  [94450a4](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/94450a4ad22d9420aaa210de2850301d9afae49d)

Reply:
Ah, none of the dark mode tickets has been merged with main so that means those NxBinaryDonutChart variables are definitely not being used by any project. I removed them here: [94450a4](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/94450a4ad22d9420aaa210de2850301d9afae49d)
[bca6dfe](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/bca6dfee2d2c6757ecaf6af7179fe665d58dee8e)